packages/hash/api/src/db/postgres/aggregation/util.ts

```diff
@@ -81,6 +81,8 @@ export const getAggregation = async (
       source_account_id = ${params.sourceAccountId}
       and source_entity_id = ${params.sourceEntityId}
       and path = ${params.path}
+    order by created_at desc  
+    limit 1
```

Comment:
Perhaps not directly relevant to this PR, but it seems some of the usages of `getAggregation` assumes that it is fallible. 
Should the query be issued with `maybeOne` instead of `one` to reflect this? I am not sure if the [`DataIntegreityError`](https://github.com/gajus/slonik#one) is that informative when it throws, as it seems the one location that throws up to a GraphQL resolver doesn't try/catch. https://github.com/hashintel/hash/blob/e3459e301ac21fe5134e403a69491c891f237fd0/packages/hash/api/src/db/postgres/aggregation/updateAggregationOperation.ts#L29-L33
I think being explicit in the error such as in  the following might be preferable.
https://github.com/hashintel/hash/blob/e3459e301ac21fe5134e403a69491c891f237fd0/packages/hash/api/src/graphql/resolvers/linkedAggregation/linkedAggregationResults.ts#L25-L32

Bu as mentioned in the start, this might not be relevant to this PR. Any thoughts?

Reply:
@thehabbos007 it might be best to create follow-up tasks for the suggestions you made, because the backend changes here are just to make the table block work correctly with the data that we have.

Reply:
Captured here: https://app.asana.com/0/1201707629991380/1201820398013202/f

Reply:
@benwerner01 for sight / thoughts

Reply:
@thehabbos007 I just want to clarify that this `getAggregation` is only used in the `updateAggregationOperation` db adapter method, where I think it's safe for us to assume that the aggregation exists because the update db method should only be called from an already initialised `Aggregation` model class instance, where the correct apollo error (you exemplify) can be thrown when `Aggregation.getEntityAggregation` returns `null` instead of an `Aggregation` instance.

And I don't think we should be throwing apollo errors in the `updateAggregationOperation` db adapter method, as the datastore implementation should be decoupled from the GraphQL API entirely.

Perhaps renaming this `getAggregation` method to `mustGetAggregation` makes it more clear that this method is assuming to find the aggregation no matter what?

Reply:
Alright, I'm not sure this is the PR to do that so I'll reuse the task @akash-joshi created to capture this.